doc/faq.rst

```diff
@@ -385,3 +385,12 @@ efficient to process for most operations. Extensive work would also be needed
 to support Pandas categorical types. Restricting input to homogeneous
 types therefore reduces maintenance cost and encourages usage of efficient
 data structures.
+
+Do you plan to implement transform for target y in a pipeline?
+----------------------------------------------------------------------------
+Currently transform only works for features X in a pipeline. There's a
+long-standing discussion about not being able to transform y in a pipeline.
+Follow on github issue #4143. Meanwhile check out
```

Comment:
Could you add the link to the discussion:
```
`#4143<https://github.com/scikit-learn/scikit-learn/issues/4143>`
```
You may need to rearrange the lines to stay below the 80 characters per line limit.

Reply:
Added link to discussion 4143